javascript-source/commands/customCommands.js

```diff
@@ -203,6 +203,21 @@
             message = $.replace(message, '(count)', $.inidb.get('commandCount', event.getCommand()));
         }
 
+        if (message.match(/\(keywordcount\s(.+)\)/g)) {
+            var input_keyword = message.match(/\(keywordcount\s(.+)\)/)[1],
+                keyword_info = JSON.parse($.inidb.get('keywords', input_keyword));
+        
+            if('count' in keyword_info){
+                ++keyword_info["count"];
+            }
+            else{
```

Comment:
Do hug the brace and put a space.

Reply:
Thanks so much for your feedback. I have updated accordingly.